client/src/pages/Translations.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import React from 'react';
+import { Tabs, Layout, Row } from 'antd';
+
+import '../css/Translations.css';
+
+const { Header } = Layout;
+const { TabPane } = Tabs;
+
+const AdminResourceManager = (props: Props): React$Element<'div'> => {
+  const { match } = props;
```

Comment:
Looks like this is leftover from the admin page, we don't need this right now (or ever? not sure actually)